Extension/CloudFoundryCLI/cloudfoundrycli.ts

```diff
@@ -65,10 +65,16 @@ if (!cfPath) {
             if (args) {
                 cfCmd.line(args);
             }
-            cfCmd.exec()
-                .fail(function (err) {
-                    tl.setResult(tl.TaskResult.Failed, '' + err);
-                });
+            
+            var cfResult = cfCmd.execSync();
```

Comment:
Only short running commands should be run synchronously. cf CLI should be run async. Please switch back to using `cfCmd.exec()`